src/server/cache.rs

```diff
@@ -306,3 +306,67 @@ async fn tiles_cache_periodic_reporter(cache: &TilesCache, metrics: &Metrics) {
     metrics.count("tiles_cache.count", cache_count);
     metrics.count("tiles_cache.size", cache_size as i64);
 }
+
+#[cfg(test)]
+mod test_tile_cache {
+
+    use super::TilesCache;
+    use crate::server::TILES_CACHE_INITIAL_CAPACITY;
+    use cadence::{NopMetricSink, StatsdClient};
+    use std::{sync::Arc, time::Duration};
+
+    #[actix_web::test]
+    async fn test_spawn_periodic_reporter() {
+        let tiles_cache = TilesCache::new(TILES_CACHE_INITIAL_CAPACITY);
+        let statsd_client = StatsdClient::builder("test", NopMetricSink).build();
+
+        // Just checks that we can spawn the periodic reporter without issues.
+        tiles_cache.spawn_periodic_reporter(Duration::from_secs(1), Arc::new(statsd_client));
```

Comment:
There are no assertions here, maybe we can check those metric counters (`tiles_cache.count` and `tiles_cache.size`) emitted inside the reporter. See [here](https://github.com/mozilla-services/contile/blob/main/src/web/test.rs#L902-L941) for an example of how we use `SpyMetricSink` for that.